index.js

```diff
@@ -49,6 +49,13 @@ function floss(options, done) {
     const app = path.join(__dirname, 'electron');
     const args = JSON.stringify(options);
 
+    let isWindows = /^win/.test(process.platform);
+    if(isWindows) {
+        if(!path.extname(options.electron)) {
+            options.electron = options.electron + ".cmd";
+        }
+    }
```

Comment:
Something a little simpler. Also could you add a note about why we're adding .cmd?
```js
if(/^win/.test(process.platform) && !path.extname(options.electron)) {
    options.electron += ".cmd";
}
```